src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedFieldSymbolBase.cs

```diff
@@ -33,21 +32,27 @@ internal abstract class SynthesizedFieldSymbolBase : FieldSymbol
                 (isStatic ? DeclarationModifiers.Static : DeclarationModifiers.None);
         }
 
+        internal virtual bool EmitDynamicAttribute
+        {
+            get { return false; }
```

Comment:
Our policy is to add abstract methods and ensure that all leaf derived types have appropriate implementation.